### ISSUE TITLE
Sample displays error for single-tenancy application.

### ISSUE THREAD

**Issue by mtbgithub** [Azure-Samples/active-directory-xamarin-native-v2#106]
I've created and configured an app registration in our portal.  For the Authentication section I've only enabled the first suggested Redirect URL (_msalXXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX://auth_) and have configured the _Supported Account Types as Accounts in this organizational directory only (XX XXXXXX only - Single tenant)_ - this is deliberate as the app is for our organisation's use only, we do not want to permit other organisations' users to access it.

Selecting the _SIGN IN_ button takes us to our organisations sign-in page and the two-factor works correctly.  However I then get an error dialog displayed which says _AADSTS50194: Application 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX' is not configured as a multi-tenant application. Usage of the /common endpoint is not supported for such applications created after '10/15/2018'. Use a tenant-specific endpoint or configure the application to be multi-tenant._ 

I am currently unable to make progress on this.  All posts I have found to date seem to recommend multi-tenancy and/or not using the /common endpoint - I'm not able to do the first and I'm already not doing the second.

Can anyone please suggest a way forward on this please.

**Comment by TiagoBrenck** [comment]
Hi @mtbgithub, did you make all the necessary changes mentioned here https://github.com/Azure-Samples/active-directory-xamarin-native-v2/tree/master/1-Basic#optional-step-3--configure-the-visual-studio-project-with-your-app-coordinates ?

Are you using the 1st sample or 2nd?

**Comment by mtbgithub** [comment]
Hi Tiago (@TiagoBrenck), thanks for your reply.  I did do all the optional steps but it didn't help.  What did finally sort this out for me and allow me to make progress was adding the TenantID to the initial construction of the PublicClientApplication object.

`        public static string TenantID = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX";

<snip>

        public App()
        {
            PCA = PublicClientApplicationBuilder.Create(ClientID)
                .WithTenantId(TenantID) // <- ADD THIS TO MAKE SINGLE-TENANT WORK
                .WithRedirectUri($"msal{ClientID}://auth")
                .Build();`

Once that was done it had no problem linking to our AAD and authenticating the credentials.

I can't remember where I found the clue to this, it was in some Python related post but it just nudged me in the right direction.

Hope it helps someone else.
